dockerfiles/platform-build-incremental/Dockerfile

```diff
@@ -54,6 +54,5 @@ RUN echo Building with $(cat ~/build_threads) threads
 RUN make -j$(cat ~/build_threads)
 
 USER root
-RUN cmake /hpcc-dev/HPCC-Platform -DUSE_PYTHON2=0
```

Comment:
doesn't look right that this was here before, but should there be a cmake line earlier (before line 54) instead? to ensure remade if something in delta has changed in cmake files.
Although I think cmake will be recalled by make anyway..

Reply:
It was here before to put in the python flag override when it was not in one of the base images, IIRC.

Reply:
make will call cmake if needed.